docs: Update Tutorial (#3411)
PrQL/prql: web/book/src/tutorial/relations.md

| 5          | 23          | Boston       |       ...       | 13.86 |
| 6          | 37          | Frankfurt    |       ...       | 0.99  |

> A relation is composed of rows, each of which is a tuple (a list) that contains
> similar columns. Each column has an unique name and a designated data type. The

Reviewer: ```suggestion
A relation is composed of rows. Each row in a relation contains
a value for each of the relation's columns. Each column in a relation has an unique name 
and a designated data type. The
```

I actually don't think we need to mention `tuple` here — we introduce it slightly later

> designated data type. The table above has columns named `invoice_id`and

Reviewer: ```suggestion
designated data type. The table above is a relation, and has columns named  `invoice_id`and
```

Reviewer: WDYT? I'm trying to find a way of making "relation" fit into this...
